grundlagen/tex/header.tex

```diff
@@ -31,6 +31,7 @@
 \usepackage{pgfpages}
 \usepackage{nicefrac}
 \usepackage{tikz}
+\usepackage{ccicons}
```

Comment:
ah, natürlich gibt's nen Paket dafür :D

Reply:
Klar, verwende ich sogar in meiner Diss für meine Paper, damit ich nicht ständig schauen muss was ich jetzt mit den Abbildungen machen darf oder nicht ;)